src/plugins/utils/tree-clusters.ts

```diff
@@ -88,30 +100,28 @@ export const metaClusterizeFlatTree = (flatTree: FlatTree, condition = defaultCl
         .filter((nodes) => nodes.length)
         .map((nodes) => ({
             nodes,
-            parents: [...new Set(nodes.map(({ parent }) => parent))],
```

Comment:
this thing seems like it was never used?

Reply:
Well, it looks like that. I searched to see if there is a use for this variable somewhere and did not find it.)